tests/nightly/test_large_array.py

```diff
@@ -128,6 +137,35 @@ def test_take():
     assert np.sum(res[-1].asnumpy() == 1) == res.shape[1]
 
 
+def test_split():
+    a = nd.arange(0, LARGE_X * SMALL_Y).reshape(LARGE_X, SMALL_Y)
+    outs = nd.split(a, num_outputs=SMALL_Y, axis=1)
+    sum = 0
+    for i, out in enumerate(outs):
```

Comment:
You may make this more pythonic:

```suggestion
    result = sum(1 for i, v in enumerate(outs) if i == out[0])
```
You also need to convert outs into outs.asnumpy() first in the line above.

Reply:
ok